.github/workflows/func.yaml

```diff
@@ -14,7 +14,7 @@ on:
           Tox version, which should be used, e.g. `<4`. If not specified, the
           latest version of tox will be installed.
         type: string
-        default: ""
+        default: "<4"
```

Comment:
We need to change the description if using `tox<4` as default. The current wording gives the wrong impression that "no user input == unspecified".